docs/format/source/format_release_notes.rst

```diff
@@ -1,6 +1,12 @@
 Release Notes
 =============
 
+2.3.0 (Upcoming)
+----------------------
+
+- All "unit" fields representing units of measurement now follow the CMIXF-12 format as described in:
```

Comment:
This is potentially code breaking change, needs more prominent notice. May be worth creating sectioning like in https://github.com/datalad/datalad/blob/master/CHANGELOG.md#0126-april-23-2020----